_templates/component/new/docs/doc.mdx.t

```diff
@@ -3,88 +3,61 @@ to: packages/<%= h.changeCase.param(name) %>/src/docs/<%= h.changeCase.pascal(na
-- [Danger](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--danger&args=#danger)
-- [Warning](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--warning&args=#warning)
-- [Info](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--info&args=#info)
-- [Light](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--light&args=#light)
-- [Dark](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--dark&args=#dark)
-- [Inverted](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--inverted&args=#inverted)
-- [Animated](?path=/docs/ui-bootstrap-<%= h.changeCase.param(name) %>--animated&args=#animated)
 
-<>
-
-### Primary
 <Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--primary"/>
-</Canvas>
-
-### Secondary
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--secondary" />
-</Canvas>
-
-### Success
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--success" />
-</Canvas>
-
-### Danger
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--danger" />
-</Canvas>
-
-### Warning
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--warning" />
-</Canvas>
-
-### Info
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--info" />
-</Canvas>
-
-### Light
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--light" />
-</Canvas>
-
-### Dark
-<Canvas>
   <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--dark" />
 </Canvas>
 
-### Inverted
+# Events
+Button will emit this events while changing state
 
-Use the `invert` prop to alternate styles 
-<Canvas>
-  <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--inverted" />
-</Canvas>
+| Event | Params | Desription |
+| ---- | ---- | ----------- |
+| Toggle | active: `boolean` | Will only raise when `toggle = true` |
 
-### Dismissible
+# Accesibility
+Change the next properties to change the default accesibility text
 
-Use the `dismissible` prop to add a closable <%= h.changeCase.param(name) %>
-<Canvas>
-  <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--dismissible" />
-</Canvas>
+| Name | Property | Default |
+| ---- | ---- | ----------- |
+| role | aria:role | `button` |
 
-### Animated
+Check the page [what is aria](/docs/docs-aria--page#what-is-aria-config) to see how to config accesibility globally for buttons.
 
-Use the `animate` prop to animate an <%= h.changeCase.param(name) %>
-<Canvas>
-  <Story id="ui-bootstrap-<%= h.changeCase.param(name) %>--animated" />
-</Canvas>
-</>
+
+# Theme
+Use the next `json` to define a new theme for the component
+
+```json
+{
+  cssClass: {
+    main: string, // main class
+    variants: { // define a custom list of variants
+      'variant1': string,
+      'variant2': string,
+      ...
+      'variantN': string,
+    },
+  },
+}
+```
+
+Check the page [what is a theme](/docs/docs-theming--page#what-is-a-theme) to see how to use this json
```

Comment:
Could you please add a new empty line at the end of the file?

Reply:
Done